students/rachel_s/session01/pp_strings2.py

```diff
@@ -0,0 +1,69 @@
+# Given a string, return a string where for every char in the original, there 
+# are two chars.
+def double_char(str):
+    newstring = ''
+    for i in str:
+        newstring = newstring + i + i
+    return newstring
+
+
+# Return the number of times that the string "hi" appears anywhere in the given 
+# string.
+def count_hi(str):
+    count = 0
+    for i in range(len(str) - 1):
+        if(str[i:i+2] == "hi"):
```

Comment:
It's really minor, but parenthesizing the entire if condition in python is not needed and actually not desired.